content/5_Plat/HLD.mdx

```diff
@@ -15,6 +15,253 @@ frequency: 1
 
 ## Tutorial
 
+In some advanced problems on trees, the problem may ask you to support the
+following operations:
+
+- Update all nodes along the path from node $x$ to node $y$ (Note that this path
+  is unique due to properties of trees).
+
+- Find the sum, maximum, minimum (or any other operation that satisfies the
+  associative property) along the path from node $x$ to node $y$.
+
+Other simpler algorithms such as difference on trees or LCAs could be used to
```

Comment:
> Other simpler algorithms such as difference on trees or LCAs

What does this mean?

Reply:
I was trying to say that if the problem only asks you to support one of the queries other algorithms could be used. Though looking up its kinda irrelevant, I removed it in the recent update.

Reply:
I was trying to say that if the problem only asks you to support one of the queries other similar algorithms could be used. Though after revisiting it I found it kinda irrelevant. I removed in the recent update.